packages/apollo-link-error/package.json

```diff
@@ -32,7 +32,8 @@
     "watch": "tsc -w -p . & rollup -c -w"
   },
   "dependencies": {
-    "apollo-link": "^1.2.1"
+    "apollo-link": "^1.2.1",
+    "apollo-link-http-common": "^0.2.2"
```

Comment:
I think `apollo-link-http-common` should be a devDependency, since we are just using it for types and tests. Just in case someone is just using a websocket transport  or absinthe.

Reply:
I really appreciate your gentle review. I will move it to devDependency. 😀